src/wq/ql/notify.py

```diff
@@ -132,27 +132,39 @@ def list_insert_tasks(task_status: Optional[str] = None):
     task_id_prefix = build_task_id_init_segment()
     find_tasks = insert_task_finder(task_status)
     response_payload = find_tasks(task_id_prefix)
-    log().info("Retrieved notification tasks to the work queue successfully")
+    try:
+        task_status == TaskStatus.FAILED.value
+    except Exception as e:
+        log().error(e)
```

Comment:
I don't think this try/catch block will have any effect? `TaskStatus` is an enum, the value field won't ever raise an exception?

Anyways, the exception we should catch is the one `find_tasks` could throw, so the code should rather be

```python
try:
    task_id_prefix = build_task_id_init_segment()
    find_tasks = insert_task_finder(task_status)
    response_payload = find_tasks(task_id_prefix)
    log().info("Retrieved notification tasks to the work queue successfully")
    return build_json_array_response_stream(response_payload)
except Exception as e:
    log().error("list_insert_tasks failed: ", e)
    raise e
```

What do you think?